repl/src/test/scala/org/apache/spark/repl/ReplSuite.scala

```diff
@@ -55,7 +55,14 @@ class ReplSuite extends SparkFunSuite with BeforeAndAfterAll {
     Main.sparkSession = null // causes recreation of SparkContext for each test.
     Main.conf.set("spark.master", master)
 
-    val in = new BufferedReader(new StringReader(input + "\n"))
+    class SlowBufferedReader(in: Reader, readLineDelay: Long) extends BufferedReader(in) {
+      override def readLine(): String = {
+        if (readLineDelay > 0) Thread.sleep(readLineDelay)
```

Comment:
"Put curly braces even around one-line conditional or loop statements" 
As suggested in https://github.com/databricks/scala-style-guide#curly-braces